api/machine_types.go

```diff
@@ -303,9 +303,11 @@ var (
 	MachineRestartPolicyAlways    MachineRestartPolicy = "always"
 )
 
+// @description The Machine restart policy defines whether and how flyd restarts a Machine after its main process exits. See https://fly.io/docs/machines/guides-examples/machine-restart-policy/.
 type MachineRestart struct {
+	// no: Never try to restart a Machine automatically when its main process exits, whether that’s on purpose or on a crash; always: Always restart a Machine automatically and never let it enter a stopped state, even when the main process exits cleanly; on-failure: Try up to MaxRetries times to automatically restart the Machine if it exits with a non-zero exit code. Default when no explicit policy is set and for Machines with schedules.
```

Comment:
We can make it show up like a real enum - https://github.com/swaggo/swag#add-a-description-for-enum-items